src/widgets/NewExpiryDateInput.js

```diff
@@ -106,40 +111,21 @@ export const NewExpiryDateInput = React.memo(
 
     const internalViewStyle = getAdjustedStyle(expiryBatchView, width, isLastCell);
 
-    // Too many TextInputs causes React Native to crash, so only
-    // truly mount the TextInput when the Cell is focused.
-    // Use TouchableWithoutFeedback because we want the appearance and
-    // feedback to resemble a TextInput regardless of focus.
-    if (!isFocused) {
-      const text = usingPlaceholder ? placeholder : expiryDate;
-
-      return (
-        <TouchableWithoutFeedback onPress={() => dispatch(focusAction(rowKey, columnKey))}>
-          <View style={internalViewStyle}>
-            <View style={expiryBatchTextView}>
-              <Text ellipsizeMode="tail" numberOfLines={1} style={textStyle}>
-                {text}
-              </Text>
-            </View>
-          </View>
-        </TouchableWithoutFeedback>
-      );
-    }
-
     // Render a Cell with a textInput.
     return (
       <View style={internalViewStyle}>
         <TextInput
+          ref={getCellRef(refIndex)}
           placeholder={placeholder}
           style={textStyle}
           value={expiryDate}
+          placeholderTextColor={placeholderColour}
           onChangeText={setExpiryDate}
-          autoFocus={isFocused}
           onSubmitEditing={onSubmit}
           onEndEditing={finishEditingExpiryDate}
-          underlineColorAndroid={dataTableColors.editableCellUnderline}
+          underlineColorAndroid="#CDCDCD"
```

Comment:
could the colour const be moved to a shared location (maybe `colors.js`)?
( and line 157 )